docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+     -
+   * - **Extractor**
+     -
+     -
+   * - :doc:`/tools/extractor` (``daml extractor``)
+     - Labs
+     -
+
+.. _libraries:
+
+Libraries
+---------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **Scala Ledger API Bindings**
+     -
+     -
+   * - ``daml codegen scala`` :doc:`CLI and generated code </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - ``bindings-scala_2.12`` :doc:`library and its public API </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - **Java Ledger API Bindings**
+     - 
+     -
+   * - ``daml codegen java`` :doc:`CLI and generated code </app-dev/bindings-java/codegen>`
+     - Stable
+     -
+   * - ``bindings-java`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.6-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.7-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.8-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-dev-archive-java-proto``
+     - Alpha
+     -
+   * - **Node.js Ledger API Bindings**
+     -
+     -
+   * - ``@digital-asset/binings-js`` :doc:`Node.js library </app-dev/bindings-js>`
+     - Stable
+     - 
+   * - **JavaScript Client Libraries**
+     -
+     -
+   * - ``daml codegen js`` :doc:`CLI and generated code </app-dev/bindings-ts/daml2js>`
+     - Stable
+     -
+   * - ``@daml/types`` :doc:`library and its public API </app-dev/bindings-ts/daml-types>`
+     - Stable
+     -
+   * - ``@daml/ledger`` :doc:`library and its public API </app-dev/bindings-ts/daml-ledger>`
+     - Stable
+     -
+   * - ``@daml/react`` :doc:`library and its public API </app-dev/bindings-ts/daml-react>`
+     - Stable
+     -
+   * - **DAML Libraries**
+     -
+     -
+   * - :doc:`The DAML Standard Library </daml/stdlib/index>`
+     - Stable
+     -
+   * - :doc:`The DAML Script Library </daml-script/api/index>`
+     - Stable
+     -
+   
+.. _dev-tools:
+
+Developer Tools
+---------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **SDK**
+     -
+     -
+   * - Windows SDK (installer_)
+     - Stable
+     -
+   * - :ref:`Mac SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :ref:`Linux SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :doc:`/tools/assistant` with top level commands
+
+       - ``--help``
+       - ``version``
+       - ``install``
+       - ``uninstall``
+     - Stable
+     -
+   * - ``daml start`` helper command and associated CLI (``daml start --help``)
+     - Stable
+     - 
+   * - ``daml deploy`` :doc:`helper command and associated CLI </deploy/generic_ledger>` (``daml deploy --help``)
```

Comment:
No stability marker, I would call this stable.

Reply:
Indeed, just an omission, thanks for spotting.